.golangci.yml

```diff
@@ -32,6 +31,10 @@ linters-settings:
     shadow: true
   misspell:
     locale: US
+  # (rkatz) - Added as we have a couple of APIs conversion (eg Istio, Kong) that convert uint->int and there's no solution
```

Comment:
this check was bumped with golangci-lint 1.64 (1.59 wasn't working properly) and is raising problems on providers (istio, kong) that use their own API.